lib/membrane/bin/action.ex

```diff
@@ -82,6 +81,16 @@ defmodule Membrane.Bin.Action do
   """
   @type stop_timer_t :: {:stop_timer, timer_id :: any}
 
+  @typedoc """
+  Terminates bin with given reason.
+
+  Termination reason follows the OTP semantics:
+  - use `:normal` for graceful termination. Allowed only when termination was already requested
+    by parent, i.e. after `c:Membrane.Bin.handle_terminate_request/2` is called
```

Comment:
Suggestion by courtesy of Grammarly:
```suggestion
  - use `:normal` for graceful termination. Allowed only when the parent already requested termination,
    i.e. after `c:Membrane.Bin.handle_terminate_request/2` is called
```